test/Dialect/ESI/bundles.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: circt-opt %s | circt-opt | FileCheck %s
```

Comment:
I think this is missing some erroring cases which tests your custom type parser for `!esi.bundle`.